bin/copyRawDataToPrm.sh

```diff
@@ -562,10 +563,12 @@ else
 		else
 			log4Bash 'INFO' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "Processing run ${filePrefix} ..."
 		fi
+
 		# shellcheck disable=SC2174
 		mkdir -m 2770 -p "${PRM_ROOT_DIR}/logs/"
 		# shellcheck disable=SC2174
 		mkdir -m 2770 -p "${PRM_ROOT_DIR}/logs/${filePrefix}/"
+		log4Bash 'INFO' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "FILEPREFIX: ${PRM_ROOT_DIR}/logs/${filePrefix}/"
```

Comment:
Als je de waarde van een variabele logt, dan graag de naam van de variabele exact overnemen.
```FILEPREFIX: ...``` Suggereert een geexporteerde environment variabele, omdat het met hoofdletters geschreven is -> ```filePrefix: ...```

Reply:
ook debug dingen, door hoofdletters te gebruiken zie je net even wat sneller je output tussen alle output die je krijgt tijdens je -l TRACE level